homeworks/vladyslav.kovalov_kovalov/3-async/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/a99b8e16e5.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="page-container">
+        <header class="header-section">
+            <h1 class="header-section__title">
+                Posts
+            </h1>
+            <div class="actions-container">
+                <form class="actions-container__filter-form" onsubmit='return false' data-post-filter>
+                    <input type="text" placeholder="Type in some text..." class="actions-container__filter-control"
+                        data-post-filter-control>
+                </form>
+                <button class="actions-container__button actions-section__button button button--sort"
+                    data-button-sort>Sort<i class="button__icon fas fa-sort-amount-up-alt"
+                        data-button-sort-icon></i></button>
```

Comment:
Пофікси форматування цього файлу. Один рядок = один тег. Також якщо починаєш переносити атрибути на новий рядок то всі атрибути мають бути перенесені. Або перенось всі або не перенось їх взагалі

```suggestion
                <button class="actions-container__button actions-section__button button button--sort" data-button-sort>
                    Sort
                    <i class="button__icon fas fa-sort-amount-up-alt" data-button-sort-icon></i>
                </button>
```